target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -63,6 +63,9 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x1000 0x440
 		ath9k_patch_mac $(mtd_get_mac_ascii cfg1 RADIOADDR1)
 		;;
+	fortinet,fap-221-c)
+		caldata_extract "art" 0x1000 0x440
```

Comment:
Please extract caldata for ath10k the same way as for ath9k, by device tree bindings.